src/transformers/testing_utils.py

```diff
@@ -1121,3 +1124,26 @@ def execute_subprocess_async(cmd, env=None, stdin=None, timeout=180, quiet=False
         raise RuntimeError(f"'{cmd_str}' produced no output.")
 
     return result
+
+
+def deep_round(obj, decimals=3):
```

Comment:
The name is badly chosen IMO since you are using it primarily to convert tensors/arrays in the test below. I'd just name it `listify_and_round` (or `nested_listify_and_round` or `recursive_listify_and_round` if you really want to highlight the fact it's recursive, "deep" does not imply that IMO)

Reply:
`deep` does imply that for me (referring to `deepcopy`: https://docs.python.org/3/library/copy.html). But if it does not for you, then `nested` is fine by me.

I don't really like `listify` as it's not part of the intent, the intent is really simplifying an object for comparison. It just happens that tensors don't compare nicely (even if rounded) so I'm changing their type as a side effect (if tensors started to allow comparison, I would stop changing their type).

- `nested_round`
- `nested_simplify`

Wdyt ?

Reply:
Let's go for `nested_simplify` then!

Reply:
Done !